### PR TITLE
Added border-box as universal box-sizing with inheritance

### PR DESCRIPTION
**Pull request by frideska**
* Version 7.0.0
* Added border-box as universal box-sizing with inheritance
* Removed old direct styling with border-box
* Added content-box directly on elements dependent on it

### CODE REVIEW

File: utilities/normalizer.scss [hunk 1]
```diff
@@ -6,11 +6,20 @@
...
   line-height: 1.15; /* 1 */
   -webkit-text-size-adjust: 100%; /* 2 */
+  box-sizing: border-box;
+}
+
+/**
+ * Reset box-sizing to get the expected layout of elements
+ */
+*, *:before, *:after {
+  box-sizing: inherit;
```
Comment (JGH153) [inline utilities/normalizer.scss]: Vi bør ikke endre denne filen da vi bør kunne oppdatere den til siste versjon. Foreslår en egen scss fil

File: utilities/box-sizing.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/**
```
Comment (JGH153) [inline utilities/box-sizing.scss]: importerer du denne filen i main?

Comment (frideska) [reply utilities/box-sizing.scss]: Vups

**Comment by frideska** [comment]
[Gi ALT box-sizing: border-box; i HDD](https://trello.com/c/FKmlaHBO/299-gi-alt-box-sizing-border-box-i-hdd)

**Pull request merged**